src/WatchDogs.FakeSource/TradeModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Bogus;
+
+namespace WatchDogs.FakeSource;
```

Comment:
Why in Fake.Source? Trade is the heart of the application. Fake.Source is an implementation detail of how we generate these Trades

Reply:
Moved to contracts